website/tantalus/signals.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from orders.models import Shift, Order
+from tantalus.models import TantalusProduct, TantalusOrderVenue
+from tantalus.services import sort_orders_by_product, get_tantalus_client, TantalusException
+
+
+@receiver(post_save, sender=Shift)
+def sync_to_tantalus(sender, instance, **kwargs):
+    """Synchronize Orders to Tantalus if a Shift is made finalized."""
+    if instance.finalized:
+        try:
```

Comment:
I think it would be good to move this whole sync code to a separate service. This way, you can for example run the same code from a management command and it is good for testing